src/content/docs/tutorials/crates-tui/main.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Main
+---
+
+We are going to make our `main` function a `tokio` entry point. Adding the `#[tokio::main]` macro
```

Comment:
Did we settle on a tense to use? I thought it was "you" not "we" for docs / tutorials.

Reply:
I reworked it to using `you`.